helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -139,7 +139,7 @@ def copy(self, default=None):
         return res
 
     def write(self, vals):
-        for ticket in self:
+        for _ticket in self:
```

Comment:
Why use a recordset loop if we are working only with vals parameter?

Reply:
I do not know. It doesn't make sense, but it's like that for Odoo 11 and 12